### ISSUE TITLE
Occtax - Ajouter meta_create_date et meta_update_date sur les relevés

### ISSUE THREAD

**Issue by DonovanMaillard** [PnX-SI/GeoNature#1851]
Bonjour, 

Sur plusieurs de mes instances, les relevés vides sont indésirables et je souhaiterais les nettoyer automatiquement via une fonction planifiée. Pour être sur de ne pas supprimer des relevés en cours de saisie (qui n'ont donc pas encore d'occurrence), je souhaiterais supprimer tous les relevés sans occurrence qui n'ont pas été créés ou modifiées au cours des 2 derniers jours. Pour ca il faudrait donc connaitre les dates de création/suppression des relevés. 

Ces champs ne sont d'ailleurs pas du tout présents dans les tables occtax, il serait peut être intéressant de les prévoir à minima au niveau du relevé et des dénombrements ?

**Comment by TheoLechemia** [comment]
Salut,

Tu peut retrouver cette info dans la table `gn_commons.t_history_actions`

**Comment by camillemonchicourt** [comment]
Oui, de mémoire on ne les avait pas mis dans les tables Occtax où on les avait enlevé au profit de la table d'historisation, pour ne pas doublonner cette information.

**Comment by DonovanMaillard** [comment]
Ok super, merci pour vos retours! J'avais totalement oublié que ça pouvait se trouver dans cette table en effet ! :)
